guide/popular-topics/common-questions.md

```diff
@@ -59,6 +59,28 @@ if (<message>.author.id === '<id>') {
 }
 ```
 
+### How do I check which role was added/removed, and for which member?
```

Comment:
> Furthermore, the question is more of a miscellaneous question than an administrative one.

I agree, this should not be placed in the Administrative section but misc (subheading of common questions, not misc. examples)